spotifyPlaylist/fetch_songs.py

```diff
@@ -0,0 +1,15 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import Session
+from c3po.db.models.song import Song
+
+engine = create_engine('Database URL') 
+engine.dialect.description_encoding = None
+session = Session(bind=engine)
```

Comment:
We are using a context manager that gets called to create sessions and commit the changes
You don't need to import `sqlalchemy` here since we have already written a context manager that handles this. Please have a look at `c3po/db/metadata.py` where it has been used.